lms/djangoapps/courseware/access.py

```diff
@@ -301,6 +302,69 @@ def check_for_staff():
     return _dispatch(checkers, action, user, descriptor)
 
 
+def _has_group_access(descriptor, user, course_key):
+    """
+    This function returns a boolean indicating whether or not `user` has
+    sufficient group memberships to "load" a block (the `descriptor`)
+    """
+    if len(descriptor.user_partitions) == 0:
```

Comment:
As @andy-armstrong suggested, can we just change this to use get_cohorted_user_partition? @jimabramson